findmoney.cpp

```diff
@@ -0,0 +1,42 @@
+#include <bits/stdc++.h>
```

Comment:
in the first line, comment add the link for the question
also in the comment write the time and space complexity of the code.